advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        lon_global = configuration['communicator'].gather(lon)
</pre><p>we are using the implicit gather root to be 0 but line 10 explicity specifies rank == 0. Would it clarify things if we were to use a root_rank=0 here as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I'm pretty sure implicit root is rank 0. So anytime I call gather, the final array only exists on rank 0. That's why I've been adding those to plotting, for progress output, etc.